cmd/apiserver_receive_adapter/main.go

```diff
@@ -34,6 +35,7 @@ func main() {
 	ctx = adapter.WithInjectorEnabled(ctx)
 
 	ctx = filteredFactory.WithSelectors(ctx,
+		auth.OIDCLabelSelector,
```

Comment:
so we have to include this, to get the proper filter?

Reply:
Yeah, otherwise the selector panics when you try and fetch it from the context